book/src/merkletree.md

```diff
@@ -16,30 +29,63 @@ So for example, suppose we have the following data in a POD:
 		dateOfBirth: 1169909384,
 		userPk: 9876543210, // target user of this POD
 		_signerPk: 1234567890, // signer of the POD
-	}
+	},
+	// ...
 }
 ```
 
 The merkletree will contain the key values from the `kvs` field.
 
-Suppose that the binary representation of the key `userPk` is `1011...`. This uniquely defines the leaf position that contains the public key of the authenticated user.
+Suppose that the binary representation of the key `userPk` is `1011...`. This uniquely defines the leaf position that contains the public key of the authenticated user. Similarly for the other key-values:
 
-![](img/merkletree1.png)
+![](img/merkletree-example-2-b.png)
```

Comment:
In the diagrams I was a bit confused by the binary representation of the keys.
For example key=4 (in base 10), should be key=100 (in base 2), and then the binary encoding in little endian is [0, 0, 1].
The reason for that is that we always write numbers with the higher digits on the left and lower digits on the right.

Reply:
oh right. I was trying to put directly the little-endian encoding to make it easier to see in the diagram why that key goes to through that path, but the `_b` suffix mixes types incorrectly; will update it to make it correct.

Reply:
done: https://github.com/0xPARC/pod2/pull/31/commits/7f3a769e07f7a21fc807c41d1b9c3435a32f07bc